src/schema/schema.js

```diff
@@ -231,21 +226,34 @@ var sortMixin = {
   type: 'object',
   properties: {
     sort: {
-      type: ['object', 'boolean'],
-      default: true,
+      default: undefined,
       supportedTypes: toMap([N, O]),
-      required: ['field', 'op'],
-      properties: {
-        field: {
+      oneOf: [
```

Comment:
Actually if `default` value is provided — this is fine as we [simply return default value](https://github.com/vega/vega-lite/blob/master/src/schema/schemautil.js#L19).